api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -65,6 +67,8 @@
 	@Before
 	public void runBeforeAllTests() {
 		conceptService = Context.getConceptService();
+		
+		executeDataSet(STANDARD_TEST_DATASET_XML);
```

Comment:
Isn't this dataset automatically executed?

Reply:
See [this](https://github.com/openmrs/openmrs-core/blob/4a0feb8da351088f25fdc4e6d324a1f277aa3410/api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java#L689) test. It runs on a test [dataset](https://github.com/openmrs/openmrs-core/blob/4a0feb8da351088f25fdc4e6d324a1f277aa3410/api/src/test/resources/org/openmrs/util/databasechange/standardTest-1.9.7-dataSet.xml)